packages/react-native/BugsnagReactNative.podspec

```diff
@@ -13,9 +13,9 @@ Pod::Spec.new do |s|
   s.author       = { "Bugsnag" => "platforms@bugsnag.com" }
   s.platform     = :ios, "7.0"
   s.source       = { :git => "https://github.com/bugsnag/bugsnag-js.git", :tag => "v#{s.version}" }
-  s.source_files = "ios/BugsnagReactNative/**/*.{h,m}",
```

Comment:
Does this need to change? The glob as it was would ensure if we put anything in a subdirectory it would be picked up, so future proofing this file.

Reply:
yep - otherwise the header files could be included twice as they're included in `ios/vendor/bugsnag-cocoa`